ir-engine/etherealengine-archive
# IR-2726 Material Component Breakup

## Summary
Breaks up the material component array into explicitly defined components

## Subtasks Checklist

## Breaking Changes

## References
closes https://tsu.atlassian.net/browse/IR-2726

## QA Steps
Switch to the material-component-breakup branch on the advanced materials repo, see also
https://github.com/theinfinitereality/eepro-advanced-materials/pull/35
